news/_posts/translation/2017-05-22-do-you-speak-prestashop-may-2017.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – May 2017 edition"
+subtitle: "The latest news from our translation project"
+date:   2017-06-22 06:09:47
+authors: [ alexeven ]
+icon: icon-world
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part to the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! Read this report to see how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2017/04/DYSpeakPS.jpg)
+
+## Project news
+
+### Top languages evolution since January
+
+A lot of words have been translated since the beginning of the year :)
+Here's a quick overview of the evolution of the most active languages over the past few months - Thanks for your involvement!
+
+
+![Crowdin Monthly banner](/assets/images/2017/06/PrestaShop_translation_evolution_2017.gif)
+
+### Deactivating Swiss-German in PrestaShop
+
+It's been a while the German for Switzerland is avalaible for Switzerland merchants. However, this language does not bring any value to merchants at the moment: Indeed, it is similar to the German language, and hasn't been localized to answer the needs of the Swiss market. Confusing.
```

Comment:
"It's been a while _that_ the German (...)" ?
Also needs to end with a br tag if you want a linefeed.